# Request
int8 remaining_rows
# true means right
bool turn_direction
geometry_msgs/Pose pose
---
# Result
geometry_msgs/Pose pose
---
# Feedback